src/java.base/share/classes/sun/nio/cs/StreamEncoder.java

```diff
@@ -252,9 +246,8 @@ private boolean isOpen() {
     private ByteBuffer bb;
     private final int maxBufferCapacity;
 
-    // Exactly one of these is non-null
+    // This must be non-null
```

Comment:
I think you can drop that comment now, it only made sense when one is non-null.

Reply:
Dropped in cbf23b9f2942a5315bc160f0f653348c0d3aeee7.